### PR TITLE
Added neve sidebar promo in the dashboard

### PR DESCRIPTION
**Pull request by girishpanchal30**
Closes https://github.com/Codeinwp/otter-internals/issues/178

### Summary
I've added the `neve` theme promo notice in the otter dashboard.
For more details here: https://github.com/Codeinwp/otter-internals/issues/178#issuecomment-2252762277

### Screenshots
https://tinyurl.com/2xmck5lk

### Checklist before the final review

- [ ] Included E2E or unit tests for the changes in this PR.
- [x] Visual elements are not affected by independent changes.
- [x] It is at least compatible with the [minimum WordPress version](https://wordpress.org/plugins/otter-blocks/).
- [ ] It loads additional script in frontend only if it is required.
- [x] Does not impact the [Core Web Vitals](https://web.dev/vitals/).
- [x] In case of deprecation, old blocks are safely migrated.
- [ ] It is usable in Widgets and FSE.
- [x] Copy/Paste is working if the attributes are modified.
- [x] PR is following [the best practices]()

### CODE REVIEW

**Comment by harshitarora-in** [comment]
Hi @girishpanchal30 It is looking good. Please make the following changes:

- [ ] When clicking on the preview theme, I am facing the following error: https://vertis.d.pr/i/d92ZoU

- [ ] Please use the updated screenshot from Figma as the speed scores are not clearly visible.

**Review by HardeepAsrani — COMMENTED** [review]
@ineagu Here, do you think we should take some precautions to avoid showing this notice if user is using any other themes by us, like Raft or Neve FSE?

**Comment by girishpanchal30** [comment]
@harshitarora-in Fixed both mentioned issues with the latest commit.

**Comment by harshitarora-in** [comment]
>@harshitarora-in Fixed both mentioned issues with the latest commit.

@girishpanchal30 Please align the image in the middle.

**Comment by girishpanchal30** [comment]
@harshitarora-in Fixed, Please check with the latest commit.

**Comment by harshitarora-in** [comment]
> @harshitarora-in Fixed, Please check with the latest commit.

Looks good! Please check the below 2 screenshots:

- https://vertis.d.pr/i/yUW2Zb (add consistent spacing)
- https://vertis.d.pr/i/qsEWTz (align it properly with consistent spacing)

**Comment by girishpanchal30** [comment]
@harshitarora-in Fixed, Please check with the latest build zip.

**Comment by harshitarora-in** [comment]
@girishpanchal30 looks good now, Thanks!

File: inc/plugins/class-dashboard.php [hunk 1]
```diff
@@ -221,17 +221,44 @@ public function enqueue_options_assets() {
...
+					),
+					'neveDashboardUrl'       => esc_url(
+						add_query_arg(
+							array(
+								'page' => 'neve-welcome',
+							),
+							admin_url( 'admin.php' )
+						)
+					),
+					'neveInstalled'          => 'neve' === get_template(),
```
Comment (HardeepAsrani) [inline inc/plugins/class-dashboard.php:261]: Just one thing, we can use `NEVE_VERSION` constant for the check.

Comment (girishpanchal30) [reply inc/plugins/class-dashboard.php]: Done! Thank you for review

**Pull request merged**